.circleci/config.yml

```diff
@@ -18,6 +18,12 @@ commands:
   unittest-app-steps:
     steps:
       - checkout
+      - run:
+          name: Install pylint
+          command: sudo pip install pylint
```

Comment:
Do you need this since the `Setup Python Environment` runs `setup_python_env.sh` which activates the venv and installs pylint? Maybe move these two new steps after the `Setup Python Environment` step?

Reply:
Good catch, thanks! I thought I'd checked to see if it was part of the requirements file, but I guess I didn't check very closely.